jest.config.js

```diff
@@ -8,5 +8,13 @@
 
 module.exports = {
   rootDir: '.',
-  projects: [...require('./jest.config.oss').projects, ...require('./x-pack/jest.config').projects],
+  projects: [
+    '<rootDir>/packages/*/jest.config.js',
+    '<rootDir>/src/*/jest.config.js',
+    '<rootDir>/src/legacy/*/jest.config.js',
+    '<rootDir>/src/plugins/*/jest.config.js',
+    '<rootDir>/test/*/jest.config.js',
+    '<rootDir>/x-pack/plugins/*/jest.config.js',
+  ],
+  reporters: ['default', '<rootDir>/packages/kbn-test/target/jest/junit_reporter'],
```

Comment:
Shouldn't it contain `preset: '@kbn/test',` ?

it was added in `jest.config.oss.js` so that coverage works.